src/Ionide.ProjInfo.ProjectSystem/Ionide.ProjInfo.ProjectSystem.fsproj

```diff
@@ -1,7 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFrameworks>net6.0</TargetFrameworks>
+    <TargetFrameworks Condition="'$(BuildNet7)' == 'true'">$(TargetFrameworks);net7.0</TargetFrameworks>
```

Comment:
The reason I kept this currently is there are two scenarios.

1. 6 is installed but 7 isn't
2. 6 & 7 are both installed.

I case 1, the tests run fine for 6.0 but in case 2 they run fine for 7.0 and not 6.0.  I'm not exactly sure how to fix this yet. We could just assume going all in on 7 though.

Reply:
this makes sense, it's just an unfortunate build matrix :(